# Procedure status registration message

# The procedure status
ProcStatus[] procedure_status_vec

---
# Response status, see ServiceResponseStatus for more information.
int8 response

# If response process failed, describe the reason.
string reason
